configure.ac

```diff
@@ -4016,6 +4016,17 @@ else
     ENABLED_BIGNUM="yes"
 fi
 
+AC_ARG_ENABLE([asn-print],
```

Comment:
I think the library default behavior should be having this off, unless wolfClu or file system or openssl compatibility are enabled?

Reply:
By default, I've enabled it for desktop and disable it otherwise.
File system and such not being available disable the APIs and the application doesn't do anything.
OpenSSL compatibility isn't needed - print dotted form instead.
The application isn't built if ENABLED_ASN_PRINT is "no".

Reply:
The problem with using a must be defined macro to disable is for people using user_settings.h or custom build it will try to be included. I would prefer the macro get defined to use it. `WOLFSSL_ASN_PRINT` defined to enabled.